### PR TITLE
[OMN-715]: Add a command pallet to the web

### PR DESCRIPTION
**Pull request by gitstart-omnivore**
## Description

Using https://kbar.vercel.app/ or react-command-pallet

Commands I think we can map:

Navigation
Go to Home (Library) g then h
Preferences
Change theme (darker) v then d
Change theme (lighter) v then l
Library
(options available when on the library page)

Archive selected item e
Remove item r
Edit item labels l
Mark item as read shift + i
Mark item as unread shift + u
Sort item in ascending order s then o
Sort item in descending order s then n
Article
(options available when on an article page)

Open original article page o
Back to library u
Increase font size +
Decrease font size -
Increase margin width ]
Decrease margin width [
Edit labels l

Here is a video that gives an idea of what the UI should look like :  https://www.loom.com/share/c12086b8136e41378032d668bbf743cb

## House Keeping
- [x] Added Loom video ?
- [x] Is Deployment Link passing ?
- [x] Have you assinged PR to yourself ?
- [x] Is Github action passing ?
- [x] Have you updated the appropriate tag ?

## Ticket link (if applicable)
- https://github.com/omnivore-app/omnivore/issues/715
- https://app.gitstart.com/clients/omnivore/tickets/OMN-715
- https://app.gitstart.com/clients/omnivore/tickets/OMN-814

LOOM VIDEO
https://www.loom.com/share/d14206ba52274c47945af90063e983c1

## Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Refactor
- [ ] Others

## Remarks

- [x] I added a new package to achieve this task.
- [x] I have updated package.json

### CODE REVIEW

File: packages/web/components/templates/homeFeed/HomeFeedContainer.tsx [hunk 1]
```diff
@@ -76,17 +77,14 @@ export function HomeFeedContainer(props: HomeFeedContainerProps): JSX.Element {
 
   const gridContainerRef = useRef<HTMLDivElement>(null)
 
-  const [shareTarget, setShareTarget] = useState<LibraryItem | undefined>(
```
Comment (jacksonh) [inline packages/web/components/templates/homeFeed/HomeFeedContainer.tsx:79]: It looks like there are a bunch of style changes in this that make it a little hard to read and will mess up the git history.

**Comment by jacksonh** [comment]
It looks like there is an issue with this when trying to perform actions on active items on the library page. To duplicate try navigating to an item, then cm+k and `Edit Item labels` it looks like sometimes there is no activeItem passed to the actionHandler.

**Comment by jacksonh** [comment]
hey @gitstart-omnivore looks like this is the error on vercel:

```
147:3  Error: React Hook "useRegisterActions" is called conditionally. React Hooks must be called in the exact same order in every component render.  react-hooks/rules-of-hooks
```

**Comment by gitstart-omnivore** [comment]
@jacksonh.

- The issue with activeItem not being passed has been fixed.
- This issue -> https://github.com/omnivore-app/omnivore/issues/815 has also been resolved in this PR.

We had to comment out many keyboard shortcuts from navigationShortcuts.ts since they were mixing up with KBar shortcuts.

**Comment by gitstart-omnivore** [comment]
Adding search support for KBar https://github.com/omnivore-app/omnivore/issues/814 in this PR as well.

**Pull request merged**